networkx/readwrite/json_graph/node_link.py

```diff
@@ -45,6 +47,24 @@ def node_link_data(G, attrs=None):
         If some user-defined graph data use these attribute names as data keys,
         they may be silently dropped.
 
+        .. deprecated:: 2.8.5
+
+           The `attrs` keyword argument will be replaced with `source`, `target`, `name`,
+           `key` and `link`. in networkx 3.???
```

Comment:
FWIW, I think it should be deprecated in 2.8.6 and removed in 3.1 (we should wait for atleast 2 releases before removing stuff)